### DISCUSSION TITLE
Why does `log_param()` not work in several files?

### DISCUSSION THREAD

**Discussion by Abner-01** [mlflow/mlflow#7596] [Q&A]
I've tried using `log_param()` in several scripts in my project, but it only works in the main program. I've tried to set the `run_id` manually in the child scripts, and I've also tried to use the nested option of logging, but the exception is the same: `mlflow.exceptions.MlflowException: Run with id=xxx not found`. The `run_id` exists, is the same as the main program, but it can't be reached nor set by the log_param() method in the child script.

**Comment by JHibbard** [comment]
Hi Abner-01,

Would you be able to share a sample of code for me to look at?
